homeassistant/components/binary_sensor/ping.py

```diff
@@ -0,0 +1,133 @@
+"""
+Tracks the latency of a host by sending ICMP echo requests (ping).
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.ping/
+"""
+import logging
+import subprocess
+import re
+import sys
+from datetime import timedelta
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.binary_sensor import (
+    BinarySensorDevice, PLATFORM_SCHEMA)
+from homeassistant.util import Throttle
+from homeassistant.const import CONF_NAME, CONF_HOST
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_ROUND_TRIP_TIME_AVG = 'round_trip_time_avg'
+ATTR_ROUND_TRIP_TIME_MAX = 'round_trip_time_max'
+ATTR_ROUND_TRIP_TIME_MDEV = 'round_trip_time_mdev'
+ATTR_ROUND_TRIP_TIME_MIN = 'round_trip_time_min'
+
+CONF_PING_COUNT = 'count'
+
+DEFAULT_NAME = 'Ping Binary sensor'
+DEFAULT_PING_COUNT = 5
+DEFAULT_SENSOR_CLASS = 'connectivity'
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=5)
```

Comment:
Can you rename this to `SCAN_INTERVAL`, then it will just be called every 5 minutes and you won't need the Throttle either.